README.md

```diff
@@ -14,17 +14,16 @@ impress.js may not help you if you have nothing interesting to say ;)
 HOW TO USE IT
 ---------------
 
-[Use the source](http://github.com/impress/impress.js/blob/master/index.html), Luke ;)
+[Use the source](http://github.com/impress/impress.js/blob/master/index.html), Luke!
```

Comment:
If we are going to keep the comments it might be worth building a real annotated documentation out of it, just like what underscore does: http://underscorejs.org/docs/underscore.html.

For a consumer, reading the source should just be encouraged with an emphasis on learning how the code was made, not that one should be [relying on internals](https://medium.com/@fagnerbrack/library-internals-do-not-exist-1ff50a30e3b0) instead of the public documented API (as it is implicit in the "Use the source, Luke").